undefined reference to `sceKernelVolatileMemTryLock'

Hello.
I'm trying to use `sceKernelVolatileMemTryLock`, but it fails to compile.
main.c:
```c
#include <pspkernel.h>
#include <pspsuspend.h>

PSP_MODULE_INFO("template", 0, 1, 1);

PSP_MAIN_THREAD_ATTR(THREAD_ATTR_USER);

int main()
{
	sceKernelVolatileMemTryLock(0, NULL, NULL);

	return 0;
}
```
result:
```
psp-gcc -I. -I/home/taba/pspdev/psp/include -I/home/taba/pspdev/psp/sdk/include -G0 -O2 -Wall -D_PSP_FW_VERSION=150  -L. -L/home/taba/pspdev/psp/lib -L/home/taba/pspdev/psp/sdk/lib  -Wl,-zmax-page-size=128  main.o  -lpspdebug -lpspdisplay -lpspge -lpspctrl -lpspnet -lpspnet_apctl -o template.elf
/home/taba/pspdev/bin/../lib/gcc/psp/11.2.0/../../../../psp/bin/ld: main.o: in function `main':
(.text.startup+0x10): undefined reference to `sceKernelVolatileMemTryLock'
collect2: エラー: ld はステータス 1 で終了しました
make: *** [/home/taba/pspdev/psp/sdk/lib/build.mak:162: template.elf] エラー 1
```

This typo is causing the problem.
https://github.com/pspdev/pspsdk/blob/e20ebe38220705abcbd6693ee2161170fb9665f1/src/user/Makefile.am#L21C141-L21C164

--- a/src/user/Makefile.am
+++ b/src/user/Makefile.am
@@ -18,7 +18,7 @@ MODMGR_OBJS = ModuleMgrForUser_0000.o ModuleMgrForUser_0001.o ModuleMgrForUser_0
 
 STDIO_OBJS = StdioForUser_0000.o StdioForUser_0001.o StdioForUser_0002.o StdioForUser_0003.o StdioForUser_0004.o StdioForUser_0005.o StdioForUser_0006.o StdioForUser_0007.o StdioForUser_0008.o StdioForUser_0009.o
 
-SUSPEND_OBJS = sceSuspendForUser_0000.o sceSuspendForUser_0001.o sceSuspendForUser_0002.o sceSuspendForUser_0003.o sceSuspendForUser_0004.o sceSuspendForUser_005.o sceSuspendForUser_0006.o
+SUSPEND_OBJS = sceSuspendForUser_0000.o sceSuspendForUser_0001.o sceSuspendForUser_0002.o sceSuspendForUser_0003.o sceSuspendForUser_0004.o sceSuspendForUser_0005.o sceSuspendForUser_0006.o
 
 SYSMEM_OBJS = SysMemUserForUser_0000.o SysMemUserForUser_0001.o SysMemUserForUser_0002.o SysMemUserForUser_0003.o SysMemUserForUser_0004.o SysMemUserForUser_0005.o SysMemUserForUser_0006.o SysMemUserForUser_0007.o SysMemUserForUser_0008.o SysMemUserForUser_0009.o
 

